Feature/us 29

fedegrios opened the pull request:
Created OrganizationDto.
Created a mapper for OrganizationDto.
Modified OrganizationBusiness.
Modified Organization Controller.

emmanuelranone reviewed (changes requested):
El desarrollo esta bien. Faltaría adjuntar capturas del funcionamiento del endpoint.

fedegrios commented:
Created OrganizationDto.
Created a mapper for OrganizationDto.
Modified OrganizationBusiness.
Modified Organization Controller.

![endpoint_organizarion_public](https://user-images.githubusercontent.com/8538976/170724645-ab719787-7330-4567-8750-441a0986765b.JPG)
